tests/testthat/tests_modelEstimation.R

```diff
@@ -0,0 +1,155 @@
+context("testing modelEstimation")
+
+test_that("modelEstimation produces correct values with default args", {
+  skip_on_cran()
+  
+  # Choptank_eList has model results already, but the model 
+  # hasn't been run since it was originally saved.
+  eList <- Choptank_eList
+  info_stale <- getInfo(eList)
+  daily_stale <- getDaily(eList)
+  sample_stale <- getSample(eList)
+  
+  info_orig <- info_stale[, 1:(which(names(info_stale) == "bottomLogQ") - 1)]
+  daily_orig <- daily_stale[, 1:(which(names(daily_stale) == "yHat") - 1)]
+  sample_orig <- sample_stale[, 1:(which(names(sample_stale) == "yHat") - 1)]
+  surfaces_orig <- NA
+  eList_orig <- mergeReport(info_orig, daily_orig, sample_orig, surfaces_orig)
+  
+  eList_modeled <- modelEstimation(eList_orig)
+  info_modeled <- getInfo(eList_modeled)
+  daily_modeled <- getDaily(eList_modeled)
+  sample_modeled <- getSample(eList_modeled)
+  surfaces_modeled <- getSurfaces(eList_modeled)
+  
+  ## INFO ##
+  
+  # columns retained in info are unaltered after running modelEstimation
+  orig_cols_info <- 1:(which(names(info_modeled) == "bottomLogQ") - 1)
+  expect_true(all(names(info_orig) == names(info_modeled[, orig_cols_info])))
+  expect_equal(sapply(info_orig, '[[', 1), sapply(info_modeled[, orig_cols_info], '[[', 1))
+  
+  # defaults show up in INFO
+  expect_equal(info_modeled[['windowY']], 7)
+  expect_equal(info_modeled[['windowQ']], 2)
+  expect_equal(info_modeled[['windowS']], 0.5)
+  
+  ## DAILY ##
+  
+  # daily data is unaltered after running modelEstimation
+  expect_equal(daily_modeled[['Date']], daily_orig[['Date']])
+  expect_equal(daily_modeled[['Q']], daily_orig[['Q']])
+  expect_equal(daily_modeled[['Julian']], daily_orig[['Julian']])
+  expect_equal(daily_modeled[['Month']], daily_orig[['Month']])
+  expect_equal(daily_modeled[['Day']], daily_orig[['Day']])
+  expect_equal(daily_modeled[['DecYear']], daily_orig[['DecYear']])
+  expect_equal(daily_modeled[['MonthSeq']], daily_orig[['MonthSeq']])
+  expect_equal(daily_modeled[['Qualifier']], daily_orig[['Qualifier']])
+  expect_equal(daily_modeled[['i']], daily_orig[['i']])
+  expect_equal(daily_modeled[['LogQ']], daily_orig[['LogQ']])
+  expect_equal(daily_modeled[['Q7']], daily_orig[['Q7']])
+  expect_equal(daily_modeled[['Q30']], daily_orig[['Q30']])
+  
+  # daily modeled values come out correctly with defaults
+  expect_equal(mean(daily_modeled[['yHat']]), 0.1189346162)
```

Comment:
I don't see why not! Originally, because I was running the model without any static, pre-modeled data, I didn't have anything to compare the results to. So I needed to hard code numbers. But yes, now that I am using Choptank_eList, I should be able to compare the newly run model results to those in that "stale" eList.